### PR TITLE
Fix Next.js Starter Guide

### PR DESCRIPTION
**Pull request by estheragbaje**
This PR provides updates and fixes to the Next.js guide.

Specifically, it adds missing imports, fixes import routes and also file formats. It also adds a redirect.

### CODE REVIEW

File: docs/guides/headless-cms/build-static-website/next.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ async function getPost(slug) {
 	try {
 		const post = await directus.request(
 			readItem('posts', slug, {
-				fields: ['*', { relation: ['*'] }],
+				fields: ['*.*', { relation: ['*'] }],
```
Comment (phazonoverload) [inline docs/guides/headless-cms/build-static-website/next.md:262]: Wrong syntax. Also, I think `relation` is the wrong key here - is it not `author`?

**Review by phazonoverload — CHANGES_REQUESTED** [review]
Looks good bar the one syntax error

File: docs/guides/headless-cms/build-static-website/next.md (hunk 1 above)
Comment (estheragbaje) [reply docs/guides/headless-cms/build-static-website/next.md]: The dot notation `*.*` is to resolve all relations. I've made an update to this.

**Comment by estheragbaje** [comment]
ESLint is throwing an error for the first check
Screenshot 2024-02-12 at 16 54 31

File: docs/guides/headless-cms/build-static-website/next.md (hunk 1 above)
Comment (phazonoverload) [reply docs/guides/headless-cms/build-static-website/next.md]: That isn't right as of the new SDK. It should be:
`fields: ['*', { '*': ['*'] }],`, but can we be more specific and just pull in the author name, as required?

Comment (br41nslug) [reply docs/guides/headless-cms/build-static-website/next.md]: `fields: ['*', { '*': ['*'] }],` is not supported make sure to only use wildcards as the final item in the chain:
- `fields=*` -> `fields: ['*']
- `fields=test.*` -> `fields: [{ 'test': ['*']}]`
- `fields=*.*` -> not supported

Comment (phazonoverload) [reply docs/guides/headless-cms/build-static-website/next.md]: Well there we go. Everybody is wrong haha

Comment (estheragbaje) [reply docs/guides/headless-cms/build-static-website/next.md]: @br41nslug The syntax `*.*` still works. Should it still work when unsupported? 🤔  

I've updated the snippet based on what you sent above ^

Comment (br41nslug) [reply docs/guides/headless-cms/build-static-website/next.md]: *not supported by the Typescript types, if you're not using typescript or are ignoring the error then the API still accepts it just fine

Comment (estheragbaje) [reply docs/guides/headless-cms/build-static-website/next.md]: Alright, this makes sense

**Comment by rijkvanzanten** [comment]
ESlint should be configured to ignore TOML files, as it doesn't know how to parse those 👍🏻 (the `#` error comes from it thinking it's a JS file)

**Comment by paescuj** [comment]
> ESlint should be configured to ignore TOML files, as it doesn't know how to parse those 👍🏻 (the `#` error comes from it thinking it's a JS file)

Was just about to fix this 😄🤝 30e58e3

**Review by phazonoverload — APPROVED** [review]

**Pull request merged**